QuantConnect.ExanteBrokerage.Tests/ExanteBrokerageSymbolMapperTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using NUnit.Framework;
+
+namespace QuantConnect.ExanteBrokerage.Tests
+{
+    [TestFixture]
+    public class ExanteBrokerageSymbolMapperTests
+    {
+        ExanteSymbolMapper SymbolMapper()
+        {
+            var supportedCryptoCurrencies = ExanteBrokerage.SupportedCryptoCurrencies;
+            var clientWrapper = new ExanteClientWrapper(ExanteBrokerageHelper.ClientOptions());
+            var symbolMapper = new ExanteSymbolMapper(clientWrapper, supportedCryptoCurrencies);
+            return symbolMapper;
+        }
+
+        [Test]
+        public void ReturnsCorrectLeanSymbol()
+        {
+            var mapper = SymbolMapper();
+
+            var symbol = mapper.GetLeanSymbol("EUR/USD", SecurityType.Forex, Market.Oanda);
```

Comment:
> Which Market should be for that ticker?

Which market is EUR/USD in Exante? I guess it's probably their own right? Meaning it would be Exante Market I believe. It's not like exante is connected to Oanda and trading to it?

Reply:
There are two sections available at the Exante terminal: Cash and Forex. The difference is explained by Exante support as follows:


> Please be informed that the "Forex" tab is intended for trading currency pairs, that is, it offers trading instruments. At the same time, the "Cash" tab (Currency) is an internal list of pairs intended for currency conversion within your account.
>
>"Currency" section are used for money conversion which is stored on your account. For example, having dollars, you can convert part of them, or whole amount in Euros, Rubles, and so on at the current market rate.
>
> In relation to your question regarding Crypto we would like to let you know that crypto is not a centralized instrument, there is no single market price provider.
We are cooperating with different exchanges who provide different rates.
> Operations on Cash (.exante) instruments do not appear in Open Positions, no PnL is calculated on them. These instruments are purely for cash conversion.
>
>Clearing on different markets is performed by different prime brokers and Tier One banks. At the same time we are not a clearing member and work through our partners. We have several counterparties for each type of instrument. Orders and deals can be viewed in the reports of exchanges and our liquidity providers in case of OTC trade. As we operate under an STP model, we don't trade against our clients and do not have a dealing desk that modifies clients' trades. We do have manual trading services for bonds and exotic financial instruments execution.
>
> Unfortunately more detailed information we disclose only confidentially, on conditions of no further disclosure. It's an international practice.

Do you still think introducing `Market.Exante` is the correct way?

Reply:
@Martin-Molinero ping

Reply:
@Martin-Molinero Martin?

Reply:
There are mappings between Exante and LEAN markets at https://github.com/QuantConnect/Lean.Brokerages.Exante/blob/54fbbebce61ed3724de5abb6fa4e10e0370f32c6/QuantConnect.ExanteBrokerage/ExanteSymbolMapper.cs#L59.

As I said previously, Exante provides its local "Cash" and "Forex" for currencies. If it's "Cash", then "exchange" is "EXANTE" (e.g. `USD/SGD.EXANTE`). If it's "Forex", then exchange is "E.FX" (e.g. `USD/SGD.E.FX`). How should it implemented then?

Reply:
@Martin-Molinero ping ⬆️